aspnet/DatePicker/Server-Side-Events.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Validation
-description: Validate the date in server side
+title: Server Side Event in ASP.NET WebForm DatePicker control | Syncfusion
```

Comment:
Add letter (s) (ASP.NET WebForms)
Control ( c should be in capital)

Reply:
renamed